### ISSUE TITLE
Implement retention period

### ISSUE THREAD

**Issue by terual** [LMS-Community/lms-stats-service#2]
I think it would be good to delete the raw data after a certain retention period (Home Assistant uses 60 days). If a system does not send an update within 60 days we can conclude the install is not active anymore, so it can be deleted to not skew the totals.

**Comment by michaelherger** [comment]
Yep, we should do that. I'll add a line to the update tasks. Should be a simple `DELETE` request.

**Comment by michaelherger** [comment]
```sql
DELETE FROM <table> WHERE UNIXEPOCH(DATETIME()) - UNIXEPOCH(lastseen) > 86400*60
```

**Comment by michaelherger** [comment]
This has been implemented long ago. Retention time is 90 days.

https://github.com/LMS-Community/lms-stats-service/blob/main/cron-cleanup/src/cleanup.ts
